contracts/other/notional/README.md

```diff
@@ -0,0 +1,35 @@
+# Notional fCash as collateral
+
+## About fCash
+Notional borrowing and lending positions are stored as [fCash](https://docs.notional.finance/notional-v2/notional-v2-basics/fcash).
+
+fCash positions can be positive or negative. If positive they are equivalent for a lending position, and are redeemable at maturity for their face value. If negative they are borrowing positions.
```

Comment:
i guess I shouldve read this first lol

Reply:
![image](https://user-images.githubusercontent.com/38806121/155304232-85eb38d8-08c9-4ed4-a83f-5a3ad995db52.png)